src/EnergyPlus/IntegratedHeatPump.cc

```diff
@@ -1240,171 +1221,155 @@ void GetIHPInput(EnergyPlusData &state)
         // ErrorsFound will be set to True if problem was found, left untouched otherwise
         VerifyUniqueCoilName(state, CurrentModuleObject, AlphArray(1), ErrorsFound, CurrentModuleObject + " Name");
 
-        state.dataIntegratedHP->IntegratedHeatPumps(DXCoilNum).Name = AlphArray(1);
-        state.dataIntegratedHP->IntegratedHeatPumps(DXCoilNum).IHPtype = "AIRSOURCE_IHP";
+        auto &ihp = state.dataIntegratedHP->IntegratedHeatPumps(CoilCounter);
+
+        ihp.Name = AlphArray(1);
+        ihp.IHPtype = "AIRSOURCE_IHP";
 
         // AlphArray( 2 ) is the water sensor node
 
-        state.dataIntegratedHP->IntegratedHeatPumps(DXCoilNum).SCCoilType = "COIL:COOLING:DX:VARIABLESPEED";
-        state.dataIntegratedHP->IntegratedHeatPumps(DXCoilNum).SCCoilName = AlphArray(3);
-        Coiltype = state.dataIntegratedHP->IntegratedHeatPumps(DXCoilNum).SCCoilType;
-        CoilName = state.dataIntegratedHP->IntegratedHeatPumps(DXCoilNum).SCCoilName;
+        ihp.SCCoilType = "COIL:COOLING:DX:VARIABLESPEED";
```

Comment:
Is there a need to store this string? Will it ever be used other than in the `ValidateComponent` call below?